### DISCUSSION TITLE
connection error.

### DISCUSSION THREAD

**Discussion by pandat1i** [ZuodaoTech/everyone-can-use-english#742] [Q&A]
安装好后使用时显示！connection error.电脑是 windows系统的，请问应该怎么解决呢？谢谢

**Comment by an-lee** [comment]
这个就是网络连不通啊。你具体是哪一步报错？

**Comment by pandat1i** [comment]
谢谢回答，在翻译或者音频转文本的时候。如图所示，谢谢哈

![2024-06-30 194332](https://github.com/ZuodaoTech/everyone-can-use-english/assets/52093026/5509fa17-e688-424f-b9e3-450a9a70e30c)
